recipe/conda_build_config.yaml

```diff
@@ -446,16 +446,17 @@ libarchive:
 # keep in sync with arrow_cpp (libarrow exists only from 10.x,
 # but make sure we have same length for zip as arrow_cpp)
 libarrow:
+  - 16
   - 15
   - 14
   - 13
   - 12
```

Comment:
Should we drop this 12 as well?

Reply:
Yeah; it must have been a race condition between save & commit on my part...
```suggestion
```